assets/chezmoi.io/docs/user-guide/password-managers/bitwarden.md

```diff
@@ -6,22 +6,29 @@ function.
 
 Log in to Bitwarden using:
 
-```console
+```shell
 $ bw login $BITWARDEN_EMAIL
 ```
 
 Unlock your Bitwarden vault:
 
-```console
+```shell
 $ bw unlock
 ```
 
 Set the `BW_SESSION` environment variable, as instructed.
 
+!!! tip "Bitwarden Session One-liner"
+    Set `BW_SESSION` automatically with:
+
+    ```shell
+    $ export BW_SESSION=$(bw {login,unlock} --raw)
+    ```
+
 The structured data from `bw get` is available as the `bitwarden` template
 function in your config files, for example:
 
-```
+```golang
```

Comment:
Remove `golang`. Same for the next blocks.

Reply:
Fixed!